addons/portal/models/res_lang.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+from odoo.http import request
+from operator import itemgetter
+
+
+class Lang(models.Model):
+    _inherit = "res.lang"
+
+    @api.model
+    def get_available(self):
+        """ Return the available languages as a list of (code, name) sorted by name. """
+        if request and request.is_frontend_multilang:
+            langs = self.search([])
+            return sorted([(lang.code, lang.url_code, lang.name) for lang in langs], key=itemgetter(2))
+        return super(Lang, self).get_available()
```

Comment:
I don't know about this without investigating... @JKE-be ?

Reply:
TODO
- replace is_frontend_multilang in is_frontend
- add a comment for the active_test removal (or better)